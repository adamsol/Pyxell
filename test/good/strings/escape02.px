
s = "\\\'\"\n\r\t\0"
c = ['\\', '\'', '\"', '\n', '\r', '\t', '\0']
print s.length
print s == c.join()
print s[:4], c[0]
print (s, 1)
print c
print c.map(_.code)
