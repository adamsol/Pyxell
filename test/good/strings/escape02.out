7
true
\'"
 \
("\\'\"\n\r\t\0", 1)
['\\', '\'', '"', '\n', '\r', '\t', '\0']
[92, 39, 34, 10, 13, 9, 0]
